lib/internal/process/nextTick.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+exports.setup = setupNextTick;
+
+function setupNextTick() {
+  const promises = require('internal/process/promises');
+  const emitPendingUnhandledRejections = promises.setup(scheduleMicrotasks);
+  var nextTickQueue = [];
+  var microtasksScheduled = false;
+
+  // Used to run V8's micro task queue.
+  var _runMicrotasks = {};
+
+  // *Must* match Environment::TickInfo::Fields in src/env.h.
+  var kIndex = 0;
+  var kLength = 1;
```

Comment:
ooooh true I should go through and do these changes since this is a diff anyways